smclib/CMakeLists.txt

```diff
@@ -36,8 +37,8 @@ endif()
 
 ament_python_install_package(${PROJECT_NAME})
 
-ament_export_dependencies(ament_cmake)
-ament_export_include_directories(include)
+ament_export_include_directories(include/${PROJECT_NAME})
```

Comment:
If you export targets, what purpose is this line?

Reply:
It's to support downstream packages that haven't yet switched over to targets.  It's a nice way to make sure we can gradually enable targets without forcing all downstream packages to switch at the same time.